app/controllers/api/storage.php

```diff
@@ -915,6 +915,12 @@
         }
 
         if (empty($output)) {
+            // when file extension is provided but it's not one of our
+            // supported outputs we fallback to `jpg`
+            if (!empty($type) && !array_key_exists($type, $outputs)) {
```

Comment:
But wouldn't this break everything else? what happened if someone tries to use `exe` as an image?

Reply:
`exe` will not have the supported `mme/type` which case is already captured above where we default back to our default image preview logo